mobile-api/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+services:
+  mongodb:
+    image: mongo:4.0
+    ports:
+      - '27017:27017'
+    restart: always
+    volumes:
+      - 'mongodb:/data/db'
+  api:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    depends_on:
+      - mongodb
+    environment:
+      - MONGODB_URL=mongodb://mongodb:27017/mobile-api
```

Comment:
I'm not exactly sure on how to access local mongodb inside a container. Also this will mainly be used for production so I don't think making an override will make much difference.